lib/queue.js

```diff
@@ -31,35 +31,40 @@ metasync.ConcurrentQueue.prototype.add = function(
       this.items.push(item);
     }
   }
+  return this;
 };
 
 metasync.ConcurrentQueue.prototype.next = function(
   item // process next item from queue
 ) {
   const queue = this;
   let timer;
-  if (queue.isOnPause) return;
-  queue.count++;
-  if (queue.timeout) {
-    timer = setTimeout(() => {
-      const err = new Error('ConcurrentQueue timed out');
-      queue.emit('timeout', err);
-    }, queue.timeout);
-  }
-  const stub = (item, callback) => callback();
-  const fn = queue.events.process || stub;
-  fn(item, () => {
-    queue.count--;
+  if (!queue.isOnPause) {
```

Comment:
this will be goog for code style and this will minimize changes so PR purpose will be clear